gtk4/src/native_dialog.rs

```diff
@@ -0,0 +1,55 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::prelude::*;
+use crate::NativeDialog;
+use crate::ResponseType;
+use std::cell::Cell;
+use std::future::Future;
+use std::pin::Pin;
+
+pub trait NativeDialogExtManual {
+    // rustdoc-stripper-ignore-next
+    /// Shows the dialog and returns a `Future` that resolves to the
+    /// `ResponseType` on response.
+    ///
+    /// ```no_run
+    /// # use gtk4 as gtk;
```

Comment:
```suggestion
```
there's no mention of gtk4 in the docs, it's mostly a crate name

Reply:
What do you mean? I think we need this line for the example to compile.

Reply:
We already assume the package is renamed from gtk4 to gtk.

Reply:
Added your change, no the tests fail because the example cannot compile.

Reply:
wait, was that part of the code and not just a comment?

Reply:
Yes. The hash just hides it in the docs:

https://doc.rust-lang.org/rustdoc/documentation-tests.html#hiding-portions-of-the-example

Reply:
aah oh I see, TIL. My bad then, please add it back :)

Reply:
Done :)